src/stateToHTML.js

```diff
@@ -247,15 +253,30 @@ class MarkupGenerator {
     return this.blocks[this.currentBlock + 1];
   }
 
-  writeStartTag(blockType) {
-    let tags = getTags(blockType);
+  writeStartTag(block) {
+    let tags = getTags(block.getType());
+
+    let attrString;
+    if (this.options.blockStyleFn) {
+      let { attributes, style } = this.options.blockStyleFn(block) || {};
```

Comment:
if we allowed `element` here too, this feature could probably replace the feature proposed by #31